explainer.md

```diff
@@ -76,14 +87,14 @@ object({})    // returns the object {}
 object(null)  // returns null
 ```
 
+For [`ref`](#struct-type-references), the same kind of type check without coercion is performed, but the exact kind of check depends on the type. This specification provides a [definition of one type system](#type-checking-for-struct-type-references), but host environments can add additional type systems.
```

Comment:
Could we introduce `ref` above as `ref(T)` so that here we could say "the same kind of type check without coercion is performed, but additionally the object is required to match `T`, in a manner described below".  Also, "host environments can add additional type systems" sounds pretty open-ended and scary (type systems don't usually compose).  Instead of making this general point in the context of `ref` and saying "type system", I'd just say "Host environments can define additional value types" as part of the general Value Types introduction.

Reply:
Indeed, that does sound scary, now that I think about it :) Changed pretty much like you suggested.